src/service/parallax-impl.js

```diff
@@ -168,16 +109,24 @@ export class ParallaxElement {
 
     /** @private {number} */
     this.offset_ = 0;
+
+    /** @private {number} */
+    this.previousScroll_ = 0;
   }
 
   /**
    * Apply the parallax effect to the offset given how much the page
```

Comment:
Update the description after "given", since now the function isn't given the delta.